vaadin-combo-box-flow-parent/vaadin-combo-box-flow-integration-tests/src/test/java/com/vaadin/flow/component/combobox/test/validation/MultiSelectComboBoxBinderValidationIT.java

```diff
@@ -0,0 +1,88 @@
+package com.vaadin.flow.component.combobox.test.validation;
+
+import com.vaadin.flow.component.combobox.testbench.MultiSelectComboBoxElement;
+import com.vaadin.flow.testutil.TestPath;
+import com.vaadin.tests.validation.AbstractValidationIT;
+import org.junit.Test;
+import org.openqa.selenium.Keys;
+
+import static com.vaadin.flow.component.combobox.test.validation.ComboBoxBinderValidationPage.*;
+
+@TestPath("vaadin-multi-select-combo-box/validation/binder")
+public class MultiSelectComboBoxBinderValidationIT
+        extends AbstractValidationIT<MultiSelectComboBoxElement> {
+
+    @Test
+    public void fieldIsInitiallyValid() {
+        assertClientValid();
+        assertServerValid();
+        assertErrorMessage(null);
+    }
+
+    @Test
+    public void required_triggerInputBlur_assertValidity() {
+        testField.sendKeys(Keys.TAB);
+        assertServerInvalid();
+        assertClientInvalid();
+        assertErrorMessage(REQUIRED_ERROR_MESSAGE);
+    }
+
+    @Test
+    public void required_changeValue_assertValidity() {
+        $("input").id(EXPECTED_VALUE_INPUT).sendKeys("foo", Keys.ENTER);
+
+        // Binder validation fails
+        testField.selectByText("bar");
+        assertServerInvalid();
+        assertClientInvalid();
+        assertErrorMessage(UNEXPECTED_VALUE_ERROR_MESSAGE);
+
+        // Binder validation passes
+        testField.deselectAll();
```

Comment:
Multi-select combo box needs to be cleared after each step, otherwise it would just select additional values.